.travis.yml

```diff
@@ -10,8 +10,9 @@ services:
 language: c
 
 before_install:
-  - git clone https://github.com/sancus-pma/sancus-main.git
-  - cd sancus-main/docker && make build SANCUS_SECURITY=$SECURITY 
+  - git clone https://github.com/Sven-C/sancus-main.git
```

Comment:
This should be reverted to upstream before merging (after sancus-step has been merged upstream (https://github.com/sancus-pma/sancus-support/pull/5))